### ISSUE TITLE
Issues cleanup

### ISSUE THREAD

**Issue by niksedk** [SubtitleEdit/subtitleedit#8231]
I've spring-cleaned the issues.

I've not done this to offend anyone (and I'm sorry if I have), but the issue list was simply just too long (and too old). 

If you really feel your issue is important and would help a lot of people, you're welcome to re-open it or create a new issue.
Be sure to explain the issue well, and make it easy to re-create.

**Comment by Purfview** [comment]
Congrats on the cleanup, and... the issues are piling up already. 😆
Maybe some active contributor would want to help with the repo management.

Btw, in the issues I like to keep only bugs or the unexpected behaviors, the feature requests I move out from the issues to Discussions>Ideas, the questions can be moved to Discussions too.

**Comment by LeonCheung** [comment]
Then where should we report bugs or the unexpected behaviors regarding a WIP feature proposed in Discussions? Going back here and opening new issues? 😆 Take the new Batch Convert feature for example, many issues will have to be opened individually in that case. 

However, except that, I do agree with you on the point of separating ideas from issues, AS LONG AS Discussions are not going to be ignored easily by devs.

**Comment by Purfview** [comment]
If feature is WIP then you can discuss of ongoing implementation in the same discussion. Discussions are more suitable for that because there in a same thread you can answer to individual posts. 

Here, in the issues it quickly gets very messy and it's hard to follow who is talking to who and about what, thats why people should post only a one issue per thread.
